internal-enrichment/team-cymru-scout/docker-compose.yml

```
@@ -0,0 +1,23 @@
+services:
+  connector-pure-signal-scout:
+    image: opencti/connector-pure-signal-scout:rolling
```

Thanks for your changes @mitchm101 but the image here won't be the correct one as the CI will take the name of your directory so, for having the correct image when deploying the connector using this file, it should be

```suggestion
    image: opencti/connector-team-cymru-scout:rolling
```

Could it be changed ?